src/internal-version.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Returns the internal AMP runtime version. Note that this is not the RTV,
+ * which is a prefix and the runtime version.
+ *
+ * TODO: Calling this function should be replaced by a compile-time constant
+ * string. For now, function's return value is compiled, not the call site.
+ *
+ * @return {string}
+ */
+export function version() {
```

Comment:
Nit: `rtv()`

Reply:
"Note that this is not the RTV, which is a prefix and the runtime version."

Reply:
This distinction is definitely not consistently made in common usage, but ok.